lib/overcommit/hook/pre_commit/ts_lint.rb

```diff
@@ -11,7 +11,7 @@ def run
       # src/file/anotherfile.ts[298, 1]: exceeds maximum line length of 140
       extract_messages(
           output.split("\n"),
-          /^(?<file>.+?(?=\[))[^\d]+(?<line>\d+).*?/
+          /^ERROR: (?<file>.+?(?=\[))[^\d]+(?<line>\d+).*?/
```

Comment:
Why is this added here, the TSLint output like the comment on line 11 so this will never match? If you're aiming at changing the message that we return,  that's possible but that should happen outside of the regex.